### DISCUSSION TITLE
Use the Github "release" feature to easily provide compiled .bin files

### DISCUSSION THREAD

**Discussion by diecknet** [tbnobody/OpenDTU#403] [Ideas]
Hello, I was wondering if there is a reason for the for the way the compiled .bin files are distributed (under Actions and so on).
Never seen that in a project on Github before. It took me a few minutes to find the actual files.
And apparently the Github Action is already prepared to create artifacts under "releases".

**Comment by madmartin** [comment]
Hi,
after reading your critics, what exactly you propose?

**Reply by diecknet** [reply]
Since the GitHub Action Workflow already supports it, the repo maintainers could click on "create release" to create a tagged release. 
[I tried it here with a fork of this repo](https://github.com/diecknet/OpenDTU/releases/tag/test1).
  
But I think it would be a bit annoying to do that manually for every build. So we could try to improve the GitHub Action Workflow to automatically create a tagged release for every commit to the main branch. 

I can take a look into that and create a PR to improve this. But I wanted to ask first, because I was not sure if there is maybe a reason for the current process.
